CHRISCHANGES.md

```diff
@@ -0,0 +1,13 @@
+# This file documents a summary of all changes made to website
```

Comment:
This file shows up in the `_site` folder after the build and would be included in the files pushed to the website. Could we create a separate doc folder and configure Jekyll to ignore that folder? Maybe we could create a dedicated change log there. We could then create a config file for Jekyll to ignore that folder. Check out this thread on the topic:
https://talk.jekyllrb.com/t/can-i-exclude-files-in-my-project-folders-from-my-site/396